src/Core/Extensible.php

```diff
@@ -18,7 +17,18 @@
  */
 trait Extensible
 {
-    use CustomMethods;
+    use CustomMethods {
```

Comment:
I've shifted from a warming-based approach to lazy-loading for methods. We should deprecate callSetOwner in favour of the new extension method registration in my most recent commit.